website/content/en/docs/scorecard/scorecard-alpha.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Scorecard (alpha)
+weight: 25
+---
+
+# operator-sdk scorecard (alpha)
```

Comment:
```suggestion

```
Because: 
<img width="487" alt="Screenshot 2020-05-22 at 17 48 40" src="https://user-images.githubusercontent.com/7708031/82690644-7e65ad00-9c54-11ea-8efc-ef502bb3fefb.png">